pop-benchmarker/src/main/java/com/microsoft/testing/popbenchmarker/MainActivity.java

```diff
@@ -0,0 +1,385 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.testing.popbenchmarker;
+
+import android.os.Build;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.crypto.RSASSASigner;
+import com.nimbusds.jose.crypto.impl.RSAKeyUtils;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Semaphore;
+
+import static com.microsoft.testing.popbenchmarker.PopUtils.ANDROID_KEYSTORE;
+import static com.microsoft.testing.popbenchmarker.PopUtils.KEYSTORE_ALIAS;
+import static com.microsoft.testing.popbenchmarker.PopUtils.getInitializedRsaKeyPairGenerator;
+import static com.microsoft.testing.popbenchmarker.PopUtils.isInsideSecureHardware;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    private MainThreadMarshaller<String> mThreadMarshaller = new MainThreadMarshaller<>();
+
+    private List<Long> mKeyGenerationTimings = new ArrayList<>();
+    private List<Long> mKeyLoadTimings = new ArrayList<>();
+    private List<Long> mSigningTimings = new ArrayList<>();
+    //private boolean mAlreadyExecuted = false;
+
+    private TextView
+            mTv_Manufacturer,
```

Comment:
Why using underscores here and below?

Reply:
Renamed in 1f55edb94315e516be892725cceb072b7c94e63d